docs/experience/longtasks.md

```diff
@@ -0,0 +1,65 @@
+title: Long Tasks
+description: Gathering information about long tasks during hard and soft page loads
+
+Experience collects information about long tasks that happen during page loads. According to the RAIL Model, tasks that are running for longer than **50ms** and blocking the main thread are considered long and are recorded by Sematext Cloud. 
+
+Long tasks can cause issues such as:
+
+* Clunky animations and scrolling affecting user experience.
+* High input latency.
+* Long delay in events handling.
+* Delayed time when your application becomes interactive.
+
+Because of that, it is crucial to see the long tasks that are running when your application handles your user requests.
```

Comment:
Because of that, it is crucial to see the long tasks that are running when your application handles your user requests.
=> Thus, it is critical to monitor long tasks executed as part of your application's handling of user requests.

Reply:
Adjusted, thanks!